src/corelibs/U2Gui/src/util/U2FileDialog.cpp

```diff
@@ -33,6 +33,59 @@
 
 namespace U2 {
 
+
+namespace {
+
+QStringList getFileNames(QWidget* parent,
+                         const QString& caption,
+                         const QString& dir,
+                         const QString& filter,
+                         const QString& selectedFilter,
+                         QFileDialog::Options options,
+                         QFileDialog::AcceptMode acceptMode,
```

Comment:
Do we really need the last 2 options? The list of parameters is already very long and we can safely use defaults for some of them that  we do not change.
We can re-introduce them back when we will have the first use-case. Until that lets not adding any boilerplate with no real need.